stdlib/format.mli

```diff
@@ -114,6 +114,14 @@ val print_float : float -> unit
 val print_char : char -> unit
 (** Prints a character in the current box. *)
 
+val print_dump_uchar : Uchar.t -> unit
+(** Prints a representation of a Unicode character in the current
+    box using only US-ASCII encoded characters according to the Unicode
+    {{:http://www.unicode.org/versions/latest/appA.pdf}notational
+    convention for code points}.
+
+    @since 4.05 *)
```

Comment:
This is not a breaking change. However if people don't want this in 4.05 I can remove it.